src/dotnetTemplate/Microsoft.PowerShell.PowerShellStandard.Template/Microsoft.PowerShell.PowerShellStandard.Template/.template.config/template.json

```diff
@@ -0,0 +1,53 @@
+{
+  "$schema": "http://json.schemastore.org/template",
+  "author": "Microsoft Corporation",
+  "classifications": [ "Common", "Console", "PowerShell", "Module" ],
```

Comment:
The module isn't a Console app.  Is classification for the resulting output?  Maybe it should be classlib?